py/fiberassign/assign.py

```diff
@@ -74,7 +74,7 @@
     ("LAMBDA_REF", "f4"),
     ("PETAL_LOC", "i2"),
     ("DEVICE_LOC", "i4"),
-    ("DEVICE_TYPE", "a3"),
+    ("DEVICE_TYPE", "S3"),
```

Comment:
Just a comment. I don't remember `aN` being a type descriptor in NumPy.  Clearly `S3` is the better choice.